pkg/engine/secrets/inspector.go

```diff
@@ -612,15 +621,31 @@ func validateCustomSecretsQueriesID(allRegexQueries []RegexQuery) error {
 func (c *Inspector) checkContent(i, idx int, basePaths []string, files model.FileMetadatas) {
 	// lines ignore can have the lines from the resolved files
 	// since inspector secrets only looks to original data, the lines ignore should be replaced
+	numRoutines := 30
 	files[idx].LinesIgnore = model.GetIgnoreLines(&files[idx])
 
 	wg := &sync.WaitGroup{}
+
 	// check file content line by line
-	if c.regexQueries[i].Multiline == (MultilineResult{}) {
+	if !c.regexQueries[i].Multiline {
 		lines := (&files[idx]).LinesOriginalData
-		for lineNumber, currentLine := range *lines {
+		startLine := 0
+		totalLineNum := len(*lines)
+		numLinesPerRoutine := totalLineNum / numRoutines
```

Comment:
add a comment that clarifies that if the file has fewer than 30 lines it will fall in the if == 0 evaluation
just to remember that `numLinesPerRoutine := totalLineNum / numRoutines` is always an integer

Reply:
modified the parallelism [here](https://github.com/Checkmarx/kics/pull/6858/commits/a353096e01ccbce4162f8d5f76ba3810b8bc0206)